nextflow_schema.json

```diff
@@ -42,8 +40,1026 @@
+                    "fa_icon": "fas fa-forward"
+                },
+                "annotation_bakta_skipgap": {
+                    "type": "boolean",
+                    "description": "Skip gap detection & annotation.",
+                    "help_text": "Skips any gene annotation found within contig assembly gaps. For more info. please refer to BAKTA [documentation](https://github.com/oschwengers/bakta). \n\n>  Modifies tool parameter(s):\n> - BAKTA: `--skip-gap`",
+                    "fa_icon": "fas fa-forward"
+                },
+                "annotation_bakta_skipori": {
+                    "type": "boolean",
+                    "description": "Skip oriC/oriT detection & annotation.",
+                    "help_text": "By default BAKTA searches for oriC/oriT genes by comparing results from Blast+ (generated by cov=0.8, id=0.8) and the [MOB-suite](https://github.com/phac-nml/mob-suite) of oriT & [DoriC](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323995/) oriC/oriV sequences. \n Annotations of ori regions take into account overlapping Blast+ hits and are conducted based on a majority vote heuristic. Region edges may be fuzzy. For more info. please refer to the BAKTA [documentation](https://github.com/oschwengers/bakta).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--skip-ori`\n",
+                    "fa_icon": "fas fa-forward"
+                }
+            },
+            "fa_icon": "fas fa-file-signature",
+            "help_text": "BAKTA  is a tool developed to annotate bacterial genomes and plasmids from both isolates and MAGs. \n\nDocumentation: https://github.com/oschwengers/bakta"
+        },
+        "annotation_prokka": {
+            "title": "Annotation: PROKKA",
+            "type": "object",
+            "description": "These parameters influence the annotation algorithm used by PROKKA.",
+            "default": "",
+            "properties": {
+                "annotation_prokka_singlemode": {
+                    "type": "boolean",
+                    "description": "Use the default genome-length optimised mode (rather than the metagenome mode).",
+                    "help_text": "By default, Prokka's --metagenome mode is used in the pipeline to improve the gene prediction of highly fragmented metagenomes.\n\nBy specifying this parameter Prokka will instead use it's default mode that is optimised for singular 'complete' genome sequences.\n\nFor more information. please check PROKKA [documentation](https://github.com/tseemann/prokka).\n\n> Modifies tool parameter(s):\n> - PROKKA: `--metagenome`",
```

Comment:
```suggestion
                    "help_text": "By default, Prokka's --metagenome mode is used in the pipeline to improve the gene prediction of highly fragmented metagenomes.\n\nBy specifying this parameter Prokka will instead use it's default mode that is optimised for singular 'complete' genome sequences.\n\nFor more information. Please check PROKKA [documentation](https://github.com/tseemann/prokka).\n\n> Modifies tool parameter(s):\n> - PROKKA: `--metagenome`",
```

Reply:
Rather: "For more information, please check PROKKA [...]"

Reply:
Changed all `PROKKA` into `Prokka` (on review branch), to keep it consistent with other docs.